Realm/RealmSwift/List.swift

```diff
@@ -105,50 +190,115 @@ public final class List<T: Object>: ListBase, SequenceType {
         }
     }
 
-    // MARK: Mutating
+    // MARK: Mutation
 
+    /**
+    Appends the given object to the end of the list.
+
+    :warning: This method can only be called during a write transaction.
+
+    :param: object An object.
+    */
     public func append(object: T) {
         _rlmArray.addObject(unsafeBitCast(object, RLMObject.self))
     }
 
-    public func append<S where S: SequenceType>(objects: S) {
-        for obj in objects {
-            _rlmArray.addObject(unsafeBitCast(obj as T, RLMObject.self))
+    /**
+    Appends the objects in the given sequence to the end of the list.
+
+    :warning: This method can only be called during a write transaction.
+
+    :param: objects A sequence of objects.
+    */
+    public func append<S: SequenceType where S.Generator.Element == T>(objects: S) {
```

Comment:
This is one of the few non-doc, non-trivial changes but it should hardly be contentious.